engine/src/main/java/com/sandboni/core/engine/finder/bcel/visitors/annotations/RunWithAnnotationProcessorDefaultImpl.java

```diff
@@ -0,0 +1,13 @@
+package com.sandboni.core.engine.finder.bcel.visitors.annotations;
+
+import com.sandboni.core.engine.sta.Context;
+import org.apache.bcel.classfile.JavaClass;
+
+public class RunWithAnnotationProcessorDefaultImpl implements RunWithAnnotationProcessor {
```

Comment:
why we need this impl if it doesn't do anything?

Reply:
foe a cleaner code when running 
new RunWithAnnotationProcessorFactory().getProcessor(value).process(jc, context);
(in com.sandboni.core.engine.finder.bcel.visitors.TestClassVisitor#visitRunWithAnnotation)